vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/parser/TestCaseParser.java

```diff
@@ -62,10 +62,26 @@ public TestCaseParser(ZephyrExporterProperties zephyrExporterProperties)
         {
             testCasesForImporting.add(new TestCase(entry.getKey(), entry.getValue().first()));
         }
+        testCasesForImporting = filterByStatuses(testCasesForImporting);
         LOGGER.info("Test cases for exporting to JIRA: {}", testCasesForImporting);
         return testCasesForImporting;
     }
 
+    private List<TestCase> filterByStatuses(List<TestCase> testCases)
+    {
+        List<TestCaseStatus> statusesForExport = zephyrExporterProperties.getStatusesForExport();
+        if (statusesForExport.isEmpty())
```

Comment:
statusesForExport sounds like we should export only test cases with such statuses, but if this is empty we export all test cases, maybe set some default value?

Reply:
Anything like `zephyr.exporter.statuses-for-export=FAILD,BROKEN,PENDING,UNKNOWN,SKIPPED,PASSED` ?